src/library/scala/collection/mutable/AnyRefMap.scala

```diff
@@ -294,6 +294,9 @@ class AnyRefMap[K <: AnyRef, V] private[collection] (defaultEntry: K => V, initi
   }
 
   /** Adds a new key/value pair to this map and returns the map. */
+  @annotation.nowarn
+  // TODO: after next re-STARR, fix the nowarn (here and in LongMap) to be more specific:
+  // @annotation.nowarn("cat=lint-multiarg-infix")  // scala/bug#12012
```

Comment:
`@nowarn("cat=dog": @nowarn)`! Otherwise, I think we just live with the build noise.

Also this one doesn't show up in the PR that deprecates them. The lint is hidden by the deprecation? Even if the deprecation is only summarized?